models/schemas/MessageDestination.yaml

```diff
@@ -0,0 +1,18 @@
+type: object
+description: "Message destination application"
```

Comment:
i dont think quotes are needed for descriptions, are they?

Reply:
Not required, but I was matching the style of most of the other schemas. Probably worth raising a ticket to standardise them?